rllib/env/multi_agent_env.py

```diff
@@ -9,7 +9,7 @@
 
 
 @PublicAPI
-class MultiAgentEnv:
+class MultiAgentEnv(gym.Env):
```

Comment:
Wow! This is great! I tried this a few moths ago and one of the gym.Env parent methods would complain due to the rewards not being floats anymore (but multi-agent Dict[str, float]s).